travis/ipa-test.sh

```diff
@@ -79,16 +98,19 @@ done
 echo "Following IPA tests are scheduled to run: "
 echo ${cert_test_file_loc}
 
-# Run ipa-tests
+trap "exit_handler" EXIT
+
+echo "Running IPA tests"
+
 ipa-run-tests \
 --ignore test_integration \
 --ignore test_webui \
 --ignore test_ipapython/test_keyring.py \
 -k-test_dns_soa \
 --verbose \
-${cert_test_file_loc}
+${cert_test_file_loc} 2>&1 | tee $TEST_LOG
```

Comment:
`ipa-run-tests` displays the test results on the terminal. Do we need to use tee?

Reply:
`ipa-run-tests` writes to `stdout` and `stderr`, which get piped to `tee`, which writes output to both the terminal _and_ `$TEST_LOG`.

Otherwise, output wouldn't be saved to a file and we'd have no way to send said file to transfer.sh (well, we could pipe directly to transfer, but they probably wouldn't like it and we wouldn't get to see output as it happens).